GameProject/Assets/TurnBasedAssets/Scripts/MouseSelection.cs

```diff
@@ -0,0 +1,56 @@
+using UnityEngine;
+
+namespace TurnBasedAssets.Scripts
+{
+    public class MouseSelection : MonoBehaviour
+    {
+        private ISelection _selection;
```

Comment:
This doesn't match your other private fields within the project. Please choose either `_camelCase` or `camelCase`.

Reply:
This is still a problem. I'm seeing `_camelCase` and `camelCase` in different files across the project. Please make it uniform.